Formula/boolector.rb

```diff
@@ -0,0 +1,58 @@
+class Boolector < Formula
+  desc "SMT solver for fixed-size bit-vectors"
+  homepage "https://boolector.github.io/"
+  url "https://github.com/Boolector/boolector/archive/refs/tags/3.2.2.tar.gz"
+  sha256 "9a5bdbacf83f2dd81dbed1e1a9f923766807470afa29b73729c947ae769d42b9"
+  license "MIT"
+
+  depends_on "cmake" => :build
+
+  resource "lingeling" do
+    url "https://github.com/arminbiere/lingeling/archive/7d5db72420b95ab356c98ca7f7a4681ed2c59c70.tar.gz"
+    sha256 "cf04c8f5706c14f00dd66e4db529c48513a450cc0f195242d8d0762b415f4427"
+  end
+
+  resource "btor2tools" do
+    url "https://github.com/boolector/btor2tools/archive/1df768d75adfb13a8f922f5ffdd1d58e80cb1cc2.tar.gz"
+    sha256 "cee19843635a15ad599424a5ad098669938afed85f6d3341e5d661cf7cd5b261"
+  end
+
+  def install
+    mkdir_p("deps/install/lib")
+    mkdir_p("deps/install/include/btor2parser")
+
+    (buildpath/"deps/lingeling").install resource("lingeling")
+    Dir.chdir("deps/lingeling")
+    system "./configure.sh", "-fPIC"
+    system "make"
+    cp("liblgl.a", "../install/lib")
+    cp("lglib.h", "../install/include")
+    Dir.chdir("../..")
+
+    (buildpath/"deps/btor2tools").install resource("btor2tools")
+    Dir.chdir("deps/btor2tools")
+    system "./configure.sh", "-fPIC"
+    system "make"
+    cp("build/libbtor2parser.a", "../install/lib")
+    cp("src/btor2parser/btor2parser.h", "../install/include/btor2parser")
+    Dir.chdir("../..")
+
+    mkdir_p("build")
+    Dir.chdir("build")
+    system "cmake", "..", "-DCMAKE_INSTALL_PREFIX=#{prefix}"
+    system "make"
+    system "make", "install"
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

Reply:
It does make sense to keep it consistent!